### DISCUSSION TITLE
Suppport other microcontroller board

### DISCUSSION THREAD

**Discussion by olimaye** [gpiozero/gpiozero#1018] [Q&A]
Hi,
what do I have to do / add in order to support other microcontroller boards than raspberry pi, e.g. Coral Dev/Mini? Can I write own Pin Factories, or which kind of low-level library support is needed? Or is there a pin factory for the Coral board, already?

**Comment by bennuttall** [comment]
You can write your own pin factory, see other implementations here: https://github.com/gpiozero/gpiozero/tree/master/gpiozero/pins

**Comment by matthijskooijman** [comment]
I ended up writing a pin factory for generically supporting any Linux board that has a gpio device interface (e.g. `/dev/gpiochip0` I think) supported by lgpio. See https://github.com/gpiozero/gpiozero/issues/1022#issuecomment-1071181630 for the code.
